.travis.yml

```
@@ -0,0 +1,29 @@
+# $ travis encrypt --add deploy.password
+deploy:
+  provider: pypi
+  distributions: sdist bdist_wheel
+  user: audreyr
+  password:
+    secure: <ENCRYPTED_TOKEN>
+  on:
+    tags: true
+    repo: andrewstillv15/Test1-Service-Repo
+    python: 3.8
+    python: 3.8
```

YAMLlint (1.37.1)

[error] duplication of key "python" in mapping

(key-duplicates)